src/vivarium/framework/results/context.py

```diff
@@ -57,6 +61,7 @@ def add_stratification(
         name: str,
         sources: List[str],
         categories: List[str],
+        excluded_categories: Optional[List[str]],
```

Comment:
Why is this a list vs a dict now?

Reply:
Not sure what you mean. This is a new thing and was never a dict

Reply:
Oh you're referring to the zoom discussion about the broken doctest?

At the highest / model spec level, `excluded_categories` is a dict because you need to specify which exclusions for which set of results.
```
# model spec
...
configuration:
  stratification:
    excluded_categories:
      cause_of_death:
        - 'stillborn'
      disability:
        - 'all_causes'
        - 'mild_child_wating'
```

But this attribute is attached to a very specific stratification (e.g. 'cause_of_death') and so is just the list to exclude.